crates/cli/src/subcommands/login.rs

```diff
@@ -79,13 +81,17 @@ async fn exec_subcommand(config: Config, cmd: &str, args: &ArgMatches) -> Result
 async fn exec_show(config: Config, args: &ArgMatches) -> Result<(), anyhow::Error> {
     let include_token = args.get_flag("token");
 
-    let identity = decode_identity(&config)?;
+    let token = if let Some(token) = config.spacetimedb_token() {
+        token
+    } else {
+        println!("You are not logged in. Run `spacetime login` to log in.");
+        return Ok(());
+    };
```

Comment:
the old version of `decode_identity` was fetching the token from the config and then producing a similar error if there wasn't one, but now we've simplified `decode_identity` to take a token as a param instead of the config. This is the only call site where we had to do this manual compensation.